view/frontend/web/js/view/payment/method-renderer/omise-offsite-truemoney-method.js

```diff
@@ -5,18 +5,16 @@ define(
         'mage/storage',
```

Comment:
We're building here on top of code that is due for quite large refactoring. Wouldn't it be less work overall to do the refactoring first? This way just creates more work. Kind of academic really since this is already done though, I guess

Reply:
Will be subject of future refactoring in some cool-off period.
Business cases are my biggest priority now.
Also, I don't see a threat of any unnecessary complexity in the future.